samples/bluetooth/peripheral_fast_pair/README.rst

```diff
@@ -192,9 +193,9 @@ Test not discoverable advertising by completing `Testing`_ and the following add
    a. Go to :guilabel:`Settings` > :guilabel:`Google` > :guilabel:`Devices & sharing` (or :guilabel:`Device connections`) > :guilabel:`Devices` > :guilabel:`Saved devices`.
    #. Verify that the paired device is appearing on the list.
 
-#. Press **Button 1** to switch to the Fast Pair not discoverable advertising show UI indication mode.
-   **LED 3** starts blinking rapidly.
-   If you want to test the Fast Pair not discoverable advertising hide UI indication mode, press **Button 1** again.
+#. Make sure that the Fast Pair Provider is in the Fast Pair not discoverable advertising show UI indication mode.
+   **LED 3** must blink rapidly.
```

Comment:
I would move this point right after the one about disconnecting Android and simplyfiy it, e.g.:
```
After disconnection, the provider will automatically switch from discoverable advertising to non-discoverable advertising show UI indication.
The LED 3 starts to blink rapidly.
```

Reply:
Maybe in line 189/190:
```
After disconnection, the provider automatically switches from the discoverable advertising to the non discoverable advertising with the show UI indication mode.
The LED 3 starts to blink rapidly.
```
and then here in line 198:
```
#. If you want to test the Fast Pair not discoverable advertising hide UI indication mode, press **Button 1**.
```

Reply:
I would do `**LED 3** is blinking rapidly.`. It starts blinking rapidly after successful pairing.